workshop/workshop-information-files/participant-information.md

```diff
@@ -0,0 +1,74 @@
+<!-- Please update this file to reflect logistics for the workshop being taught. -->
+
+Find helpful information related to health and safety, the workshop location, and more.
```

Comment:
This seems like an odd first sentence?

Reply:
It's what we've been [using](https://alexslemonade.github.io/2023-june-training/workshop/participant_information.html), but definitely can tweak it for this template!